x-pack/test/security_api_integration/tests/session_idle/cleanup.ts

```diff
@@ -18,6 +18,7 @@ import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ getService }: FtrProviderContext) {
```

Comment:
__question:__ would you mind also doing this for `session_invalidate` and `session_concurrent_limit` suites? Just in case. And we'll also need to backport this to all active branches (except for 7.17) to increase the chance of catching it.

Reply:
Done here: https://github.com/elastic/kibana/pull/193279